component/service/src/main/java/org/exoplatform/social/rest/impl/space/SpaceRestResourcesV1.java

```diff
@@ -182,6 +183,7 @@ public Response getSpaces(@Context UriInfo uriInfo,
         space.setHasBindings(spaceBindingService.isBoundSpace(space.getId()));
         
         SpaceEntity spaceInfo = EntityBuilder.buildEntityFromSpace(space, authenticatedUser, uriInfo.getPath(), expand);
+        spaceInfo.setIsUserBound(spaceBindingService.countUserBindings(space.getId(), authenticatedUser) > 0);
```

Comment:
```suggestion
        if (space.hasBindings()) {
          spaceInfo.setIsUserBound(spaceBindingService.countUserBindings(space.getId(), authenticatedUser) > 0);
        }
```

Reply:
If space doesn't has binding, no need to compute `IsUserBound` because it will be always `false`

Reply:
Aaah yes !! Thanks @boubaker